src/components/header.tsx

```
@@ -30,6 +29,16 @@ function Header(): React.ReactNode {
           >
             Spades
           </a>
+          <a
+            href="https://gbx.io"
+            className="text-gray-400 hover:text-white transition-colors text-sm"
+            target="_blank"
```

Biome (1.9.4)

[error] Avoid using target="_blank" without rel="noreferrer".

Opening external links in new tabs without rel="noreferrer" is a security risk. See the explanation for more details.
Safe fix: Add the "noreferrer" to the existing attribute.


(lint/a11y/noBlankTarget)